cellengine/resources/fcs_file.py

```diff
@@ -137,23 +147,23 @@ def events(self):
     def events(self, events):
         self._events = events
 
-    def get_events(self, **params):
+    def get_events(self, **kwargs):
         """
         Fetch a DataFrame containing this file's data.
 
         Args:
-            params (Dict): keyword arguments of form:
-                compensatedQ (bool): If true, applies the compensation
+            **kwargs:
+                - compensatedQ (bool): If true, applies the compensation
                     specified in compensationId to the exported events. For TSV
                     format, the numerical values will be the compensated values.
                     For FCS format, the numerical values will be unchanged, but the
                     file header will contain the compensation as the spill string
                     (file-internal compensation).
-                compensationId (str): Required if populationId is specified.
+                - compensationId (int): Required if populationId is specified.
```

Comment:
int or str

(int is only the special types UNCOMPENSATED, FILE_INTERNAL, the others are strings)